### PR TITLE
feat: add base64 endpoint

### PR DESCRIPTION
**Pull request by SS51219**
This is Jorge from Mexico.

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -12,6 +13,11 @@ function respondHello (req, res) {
   res.end(JSON.stringify({ msg: 'hello' }))
 }
 
+function respondBase64 (req, res) {
+  const phrase = req.url.replace(/^\/b64\//, '')
+  res.end(JSON.stringify({ b64: Buffer.from(phrase).toString('base64') }))
+}
+
```
Comment (SS51219) [inline server.js:20]: This code is added from branch "base64"

File: server.js [hunk 2]
```diff
@@ -4,6 +4,7 @@ const PORT = process.env.PORT || 3000
 
 const server = http.createServer((req, res) => {
   if (req.url === '/') return respondHello(req, res)
+  if (req.url.match(/^\/b64\//)) return respondBase64(req, res)
```
Comment (SS51219) [inline server.js:7]: This code is added from branch "base64"

File: test.js [hunk 3]
```diff
@@ -15,6 +15,15 @@ tape('should respond hello', (t) => {
...
 
+tape('should respond b64', (t) => {
+  jsonist.get(`${urlBase}/b64/hello`, (err, body) => {
+    if (err) t.error(err)
+
+    t.equal(body.b64, 'aGVsbG8=')
+    t.end()
+  })
+})
+
```
Comment (SS51219) [inline test.js:26]: This code is added from branch "base64"

**Comment by Star0112** [comment]
I made repo on my git and finished the merge.
but my repo is deleted from the git support team.
because I cloned your source and upload to my git.

**Pull request closed without merging**